### PR TITLE
DOCS-15470 add can write v1.0

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCS-15470-add-canWrite-v1.0/reference/api/progress/)

[JIRA](https://jira.mongodb.org/browse/DOCS-15470)

### CODE REVIEW

**Review by jmd-mongo — APPROVED** [review]
Thanks for the changes, @davemungo! This one LGTM!

Thanks,
Joe

**Comment by davemungo** [comment]
@adizaimi when you have a moment would you take a look at this docs update? Thanks

File: source/includes/api/tables/progress-response.rst [hunk 1]
```diff
@@ -18,6 +18,12 @@
        request will succeed. This also means that the initial sync has
        completed and is applying change events.
 
+   * - ``canWrite``
+     - boolean
+     - If ``true``, indicates that :ref:`commit <c2c-api-commit>` has
+       completed and that it is possible to write to the destination
```
[the remark this answers is not in the document]
Comment (davemungo) [reply source/includes/api/tables/progress-response.rst]: Done

**Review by davemungo — COMMENTED** [review]
@adizaimi  thanks for the feedback, this is ready for another pass

**Review by adizaimi — APPROVED** [review]
LGTM!

**Pull request merged**